### PR TITLE
Update ScaledJob migration page

### PR DESCRIPTION
**Pull request by TsuyoshiUshio**
I updated the ScaledJob documentation for v2 release. 

# What I've done 
- Add sample of V2 ScaledJob
- Add migration guide for V2

### CODE REVIEW

File: content/docs/2.0/migration.md [hunk 1]
```diff
@@ -55,10 +56,16 @@ spec:
...
-
 ### Scaling of Jobs
 
-TODO description
+In order to scale `Jobs` with KEDA v2, you need to do only a few modifications to existing v1 `ScaledObjects` definitions, so they comply with v2:
+- Change the value of `apiVersion` property from `keda.k8s.io/v1alpha1` to `keda.sh/v1alpha1`
+- Change the value of `kind` property from `ScaledObject` to `ScaledJob`
+- Remove property `spec.scaleType`
+- Remove properties `spec.cooldownPeriod` and `spec.minReplicaCount`   
+- Rename property `spec.scaleTargetRef.deploymentName` to `spec.scaleTargetRef.name`
```
Comment (zroubalik) [inline content/docs/2.0/migration.md:66]: `spec.scaleTargetRef.name` is not being used with ScaledJobs, am I right?

File: content/docs/2.0/migration.md [hunk 2]
```diff
@@ -87,8 +94,26 @@ spec:
 
 **Example of v2 ScaledJob**
 
-TODO
-
+```yaml
+apiVersion: keda.sh/v1alpha1
```
Comment (zroubalik) [inline content/docs/2.0/migration.md:98]: would you mind adding those helper comments on the changed/added/deleted lines?

eg ` #  <--- Property value was changed`

@jeffhollan like them a lot 😄

**Review by zroubalik — CHANGES_REQUESTED** [review]
Thanks, looking good, minor nits

File: content/docs/2.0/migration.md (hunk 1 above)
Comment (TsuyoshiUshio) [reply content/docs/2.0/migration.md]: Sorry, Forgot to remove. Done.

File: content/docs/2.0/migration.md (hunk 2 above)
Comment (TsuyoshiUshio) [reply content/docs/2.0/migration.md]: Thanks! Done.

**Review by tomkerkhove — APPROVED** [review]

**Review by zroubalik — APPROVED** [review]

**Pull request merged**